### PR TITLE
quick_fix for apk real name

### PR DESCRIPTION
**Pull request by superpoussin22**
### Describe the Pull Request

```
Display correct real name for apk if application name is not in app_name string  
also correct display in pdf 
```

### Checklist for PR

- [X] Run MobSF unit tests (http://your-mobsf-@ip:8000/tests/ or python3 manage.py test).
- [X] Tested Working on Linux, Mac, Windows, and Docker
- [X] Make sure build is passing on your PR. [Build Status](https://travis-ci.com/MobSF/Mobile-Security-Framework-MobSF/pull_requests)
- [X] Coding style (indentation, etc)

### CODE REVIEW

**Comment by ajinabraham** [comment]
@superpoussin22 mind resolving conflicts?

File: StaticAnalyzer/views/android/static_analyzer.py [hunk 1]
```diff
@@ -548,8 +549,9 @@ def get_app_name(app_path, app_dir, tools_dir, is_apk):
     
     data = ""
     if is_apk:
-        output_dir = os.path.join(app_dir, "apktool_out")
-        strings_file = os.path.join(output_dir, "res/values/strings.xml")
+        a = apk.APK(app_path)
```
Comment (ajinabraham) [inline StaticAnalyzer/views/android/static_analyzer.py]: Is this always guaranteed to return a real name.?

Comment (superpoussin22) [reply StaticAnalyzer/views/android/static_analyzer.py]: yep all the test I have done also with apk which did not use app_name were Ok

Comment (superpoussin22) [reply StaticAnalyzer/views/android/static_analyzer.py]: for me yes

**Pull request merged**